feature-libs/checkout/b2b/occ/config/default-occ-checkout-b2b-config.ts

```diff
@@ -1,12 +1,22 @@
+// We need these imports for augmentation of OccEndpoints to be picked up
+import { CheckoutOccEndpoints } from '@spartacus/checkout/base/occ';
 import { OccConfig } from '@spartacus/core';
```

Comment:
Inline property assignments in components : https://github.com/SAP/spartacus/pull/14165#discussion_r750886791